Makefile

```diff
@@ -19,7 +19,7 @@ build: hardcoded/hardcoded.go
 test: build generate $(PKGS) js-tests
 
 js-tests:
-	cd test/js && npm install && npm test
+	cd test/js && rm -rf node_modules && npm install && npm test
```

Comment:
ahh i see... I think it's fine to leave as is, and if it slows things down significantly we can try just removing node_modules/swagger-test